spec/osut_tests_spec.rb

```diff
@@ -1298,4 +1298,251 @@ module M
     expect(mod1.overlaps?(wall.vertices, glazing.vertices)).to be(true)
     expect(mod1.status.zero?).to be(true)
   end
+
+  it "checks subsurface insertions on tilted surfaces" do
+    # Examples of how to harness OpenStudio's Boost geometry methods to safely
+    # insert subsurfaces along rotated/tilted/slanted host/parent/base
+    # surfaces. First step, modify SEB.osm model.
```

Comment:
The end result of the basic tests:
<img width="1307" alt="seb_fen" src="https://user-images.githubusercontent.com/98104985/233791729-8d615e01-1789-4f2d-9800-f2f81958352f.png">
The intention is to support sub surface additions, regardless of surface azimuth or tilt, and independently of host surface polygon attributes (e.g. concave vs convex, regular vs irregular).